packages/react/package.json

```diff
@@ -60,5 +60,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "tailwind-merge": "^1.12.0"
```

Comment:
idk how you feel about this? this is probably the more sketchier thing

Reply:
Do you mean the fact that this is the only "real" dep, or that you find `tailwind-merge` sketchy?

Reply:
tailwind-merge is great but always skeptical about bundling deps that aren't core to the package 🤔 making it a peerDep also doesn't really make sense though so i guess it's fine?

Reply:
Yeah, definitely doesn't make sense as a peerDep. I mean it is *kinda* core to the package, at least if we consider functional styles as core to the package 🤷